app/MicroserviceModule.scala

```diff
@@ -62,7 +62,7 @@ class MicroserviceModule(val environment: Environment, val configuration: Config
     bindProperty("des.environment", "des.environment")
     bindProperty("des.authorizationToken", "des.authorization-token")
     bindProperty("auth.stride.role", URLDecoder.decode(_, "utf-8"))
-
+    bindProperty("inactive-relationships.show-last-days")
```

Comment:
what if we make it typed as Duration? then we don't have to parse string on our own?

Reply:
Ok like bindDurationProperty

Reply:
👍 I got it